### PR TITLE
Add built-in capabilities to the daemon

### PR DESCRIPTION
**Pull request by zyga**
We want to start validating capability types as they are being added. To do that though, I want to have some temporary built-in capability types so that it's easier to experiment. Later on those built-in capability types will be loaded from storage so the concept is long-lived.

### CODE REVIEW

File: caps/capabilities.go [hunk 1]
```diff
@@ -95,6 +99,16 @@ func NewRepository() *Repository {
 	}
 }
 
+// AddBuiltInTypes adds all built-in types to the repository
+// If any of the additions fail the function panics.
+func (r *Repository) AddBuiltInTypes() {
```
Comment (niemeyer) [inline caps/capabilities.go]: Feels like this shouldn't be part of the repository API. I suggest a top-level along the lines of:

    func LoadBuiltIn(r *Repository) {
            ...
    }

In the future we'll be moving these built-ins out of the binary altogether, and the repository API still won't care.

File: caps/capabilities.go [hunk 2]
```diff
@@ -95,6 +99,16 @@ func NewRepository() *Repository {
...
+	for _, t := range builtInTypes {
+		if err := r.AddType(t); err != nil {
+			panic(err.Error())
```
Comment (niemeyer) [inline caps/capabilities.go]: Doesn't feel like this should be a panic. The fact an arbitrary repository has conflicting types is a very plausible scenario. I'd suggest erroring out from LoadBuiltIn. The call site, though, may decide it's a panic situation to get an error when loading built-ins into an empty repository.

**Comment by niemeyer** [comment]
LGTM assuming the suggested tweaks.

**Comment by zyga** [comment]
Both corrected, please ack and land

**Comment by mvo5** [comment]
:+1:

**Pull request merged**